### PR TITLE
Fix dev attribute. Issue #27

### PR DESCRIPTION
**Pull request by runarb**

### CODE REVIEW

**Review by digitalsadhu — COMMENTED** [review]
Does this support both <f-broadcast dev> and <f-broadcast=true> ? Code suggests it only supports the latter but maybe the web apis smooth this over and actually both cases work?

**Comment by runarb** [comment]
I have no idea tbh. Would have helped if there were any tests around 😬

**Comment by digitalsadhu** [comment]
Tests are something I’m working on as part of the refactor I’m doing.

File: packages/broadcast/component.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export class FabricBroadcast extends FabricWebComponent {
     const existing = this.shadowRoot.querySelector('#broadcast-toast');
 
     // Generate url
-    const dev = Boolean(this.getAttribute('dev'));
+    const dev = this.getAttribute('dev') != null;
```
Comment (runarb) [inline packages/broadcast/component.js:17]: According to https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes boolean attributes should not have values "true" or "false". I think this should work in line with the spec then.

**Pull request closed without merging**